app/data/private/key_test.go

```diff
@@ -1,21 +1,39 @@
-package compat_test
+package private
```

Comment:
can this be:

```suggestion
package private_test
```

i think it could mean:

a) moving the relevant assertions of `TestKeyID` into `TestNewKey`
b) using `private.NewKey` instead of `private.keyID` during `TestKeyID`